Controller/ProductVariantAttributeController.php

```diff
@@ -0,0 +1,202 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\ProductBundle\Controller;
+
+use FOS\RestBundle\Controller\Annotations\Delete;
+use FOS\RestBundle\Controller\Annotations\Get;
+use FOS\RestBundle\Controller\Annotations\Post;
+use FOS\RestBundle\Routing\ClassResourceInterface;
+use Sulu\Bundle\ProductBundle\Product\ProductLocaleManager;
+use Sulu\Bundle\ProductBundle\Product\ProductManagerInterface;
+use Sulu\Bundle\ProductBundle\Product\ProductVariantAttributeManager;
+use Sulu\Component\Rest\ListBuilder\Doctrine\DoctrineListBuilderFactory;
+use Sulu\Component\Rest\ListBuilder\ListRepresentation;
+use Sulu\Component\Rest\RestController;
+use Sulu\Component\Rest\RestHelperInterface;
+use Sulu\Component\Security\SecuredControllerInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * This controller makes adding and removing of product variants accessible through a REST API.
+ */
+class ProductVariantAttributeController extends RestController implements ClassResourceInterface, SecuredControllerInterface
+{
+    protected static $productEntityName = 'SuluProductBundle:Product';
```

Comment:
Unfortunately the list builder depends on this naming. In the (hopefully) near future it's planned to use Sulu-Persistence and replace all the entity names here, with the correct (inheritable) entity namings